README.haiku

```diff
@@ -19,22 +19,38 @@ The build procedure is completely standard:
   make
   make install
 
-Make perl executable and create a symlink for libperl:
+=head1 NOTABLE DIFFERENCES
 
-  chmod a+x /boot/common/bin/perl
-  cd /boot/common/lib; ln -s perl5/5.35.2/BePC-haiku/CORE/libperl.so .
+=over 4
 
-Replace C<5.35.2> with your respective version of Perl.
+=item *
 
-=head1 KNOWN PROBLEMS
+Perl cannot be compiled with -fstack-protector (fixed in perl 5.32)
 
-The following problems are encountered with Haiku revision 28311:
+=item *
 
-=over 4
+Perl cannot be compiled with threading support ATM.
 
 =item *
 
-Perl cannot be compiled with threading support ATM.
+Haiku changed its filesystem hierarchy between BeOS/first Haiku versions and more recent ones.
+Haiku has a system utility "finddir" which is used to returns these paths, and
+it is now used in haiku hints (starting perl 5.32).
+
+=item *
+
+Haiku file system has a limitation and does not store the access time.
+When using "stat", it will return always the local time,
+(like if "stat" was considered as an access).
+It is not the common implementation, but neither not considered false for POSIX.
```

Comment:
I don't understand this sentence.  I think you're saying it is legal in POSIX, but not what one would expect. @Grinnz @jmacdotorg do you have good phrasing for this?